pkg/vm/stack.go

```diff
@@ -152,6 +152,9 @@ type Stack struct {
 	top  Element
 	name string
 	len  int
+
+	seen map[StackItem]int
```

Comment:
I'm thinking about renaming it into something more meaningful. `itemRefs` maybe? At the moment it's not clear at first glance who had `seen` what here. The other PR you have with `seen` is OK wrt to this, because it uses it locally, but here it's an element of a struct. Not a blocker, but something to think about.

Reply:
Essentially it's an item reference counter.

Reply:
Yeah, renamed it to `itemCount`.